client/src/components/USBController.tsx

```diff
@@ -21,6 +21,10 @@ const USBController = () => {
     return <Text>Connecting…</Text>
```

Comment:
Please change this and the other `<Text>` to `<Button>` elements as well. 

😉 This would all be more obvious if/when this component is renamed as per: https://github.com/votingworks/vxmail/issues/110